lib/devcycle-ruby-server-sdk/api/devcycle_api.rb

```diff
@@ -36,13 +36,38 @@ def initialize(sdkKey, dvc_options = DVCOptions.new, &initialize_callback)
       nil
     end
 
+    def close
+      if @dvc_options.enable_cloud_bucketing
+        @logger.info("Cloud Bucketing does not require closing.")
+        return
+      end
+      if @localbucketing != nil
+        if !@localbucketing.initialized
+          @logger.info("Awaiting client initialization before closing")
+          while !@localbucketing.initialized
+            sleep(0.5)
+          end
+        end
+        @localbucketing.close
+        @logger.info("Closed DevCycle Local Bucketing Engine.")
+      end
+
+      if @event_queue != nil
+        @event_queue.close
+        @logger.info("Closed DevCycle Event Batching")
+      end
+
+      @logger.info("Closed DevCycle Client.")
+      true
```

Comment:
the other sdks don't return a boolean indicating success for `close` so we probably don't want to do that here either

Reply:
This was just a testing artifact; will remove.